app/pb_kits/playbook/pb_currency/docs/_currency_small.jsx

```diff
@@ -3,24 +3,27 @@ import { Currency } from '../../'
 
 const CurrencySmall = () => {
   return (
-    <div>
+    <>
```

Comment:
Since we use bootstrap and may have other styles which could affect layout in our main application, this would potentially be a breaking change. I would suggest leaving the `<div>` in place here though we are not against the use of React fragments in new kits.

Reply:
OK! I thought since it was just in the documentation, it would not be a breaking change. I'll change it and keep it in mind for the future.